runtime/include/http_session.h

```diff
@@ -54,9 +54,13 @@ struct http_session {
 	size_t                  response_buffer_written;
 	struct tenant          *tenant; /* Backlink required when read blocks on listener core */
 	uint64_t                request_arrival_timestamp;
+	uint64_t                request_downloaded_timestamp;
+	uint64_t                response_takeoff_timestamp;
```

Comment:
What does takeoff mean in this context?

Reply:
It means response sending started. The naming might not be very successful. But I am measuring request receival and response sending separately. So, each has begin and end timestamps.